docs/DeveloperWorkflow/automated_checks.md

```diff
@@ -0,0 +1,38 @@
+_DeveloperWorkflow:
+
+Automated Checks for pull requests
+==================================
+
+In order to maintain the code quality and coverage of our repositories,
+keploy deploys a series of tools. These tools include our
+Continuous Integration Setup that runs a complete test suite, Automated
+Code quality checks as well as Coverage tracking tools.
+
+This section focuses on how these automated tests are set up and how they can
+be configured on our GitHub repositories. Currently only the
+following checks are available in our `keploy`_
+repository.
+
+Continuous Integration
+----------------------
+
+Keploy uses `GitHub Actions`_ for Continuous Integration. GitHub
+Actions creates an environment based on the Operating System of your
+choice and runs our test suite. This Continuous
+Integration script is triggered by every pull request and only passes
+when all the tests run successfully. 
+
+.. figure:: ../images/all checks.png
+   :alt: GitHub pull request Checks
+   :align: center
+   :width: 600px
+
+   GitHub pull request Checks
+
+-  In the case where tests fail, we can debug the problem from going
+   through the console output as displayed here.
+
+.. figure:: ../images/check fails.png
```

Comment:
In the Preview images are not rendering.

Reply:
Changes with my images link but once you merge this PR i can change the link with keploy repo

Reply:
![image](https://user-images.githubusercontent.com/67006255/173501838-7a5b3653-14b2-469e-929a-35075b3f4dcf.png)
Like this